contracts/Bridge.sol

```diff
@@ -30,12 +30,11 @@ contract Bridge is Adminable {
     bytes32[] children; // unordered list of children below this node
   }
 
-  bytes32 public constant GENESIS = 0x4920616d207665727920616e6772792c20627574206974207761732066756e21;
+  bytes32 constant GENESIS = 0x4920616d207665727920616e6772792c20627574206974207761732066756e21;
 
-  uint256 public genesisBlockNumber; 
```

Comment:
why we added this in a first place?

Reply:
🤷‍♂️ @eezcjkr you know why?

Reply:
To make it easier for the node to get this number. At first we calculated an estimate of this number and did not work for fresh chains. If we merge this the node will break, good catch.

Reply:
i've put it back